secure_storage_manager.go

```
@@ -119,6 +120,59 @@ type secureStorageManager interface {
 
 var credentialsStorage = newSecureStorageManager()
 
+// Helper fast-paths to minimize lease contention for common operations.
```

I'm concerned by all the errors we swallow up from these functions as opposed to handling them.

If there are certain we want to whitelist/swallow let's make that explicit -- at callsites or in this layer -- and fail the rest of them.

---

I got rid of these, and moved the relaxed version of get to the manager itself